servers/visual_server.cpp

```diff
@@ -2411,6 +2411,18 @@ VisualServer::VisualServer() {
 	GLOBAL_DEF("rendering/quality/depth_prepass/disable_for_vendors", "PowerVR,Mali,Adreno,Apple");
 
 	GLOBAL_DEF("rendering/quality/filters/use_nearest_mipmap_filter", false);
+
```

Comment:
batching so far is only for GLES2 (And will likely always be), so I suggest using rendering/gles2/anythinghere

Reply:
Done.